contrib/FreeBSD/README

```diff
@@ -0,0 +1,46 @@
+This folder contains sopel and sopel-default.cfg designed to be distributed by third parties such as FreeBSD.
+
+`sopel-default.cfg` is a default configuration file for sopel. The -default prefix is mandatory and will be used as a 'profile'. The profiles may be changed with the variable `sopel_profiles` in `/etc/rc.conf`. For example:
+
+```sh
+sysrc sopel_profiles="profile1 profile2 profile3"
+```
+
+The configuration files should be `${sopel_confdir}/${sopel_prefix}profile1.cfg`, `${sopel_confdir}/${sopel_prefix}profile2.cfg` and `${sopel_confdir}/${sopel_prefix}profile3.cfg`. The variable `sopel_confdir` is the default configuration directory (by default `/usr/local/etc`). `sopel_prefix`is the prefix for all files (by default `sopel-`).
+
+The service must be installed in `/usr/local/etc/rc.d`:
+
+```sh
+cp sopel /usr/local/etc/rc.d
```

Comment:
Consider naming the `sopel` script `sopel.sh` or `sopel-freebsd.sh` for clarity, and stripping that off here?

Reply:
I name the service file as such, without extension, because FreeBSD init does not need the .sh extension, and other services do not use that extension

Reply:
Right, I'm not suggesting naming the service `sopel.sh`, just naming the script one of the above in the repo, and having the command here be `cp sopel.sh /usr/local/etc/rc.d/sopel` to install it without the extension. Not a blocker if you don't like it.

Reply:
No problem, I will change the name